pallets/vesting/src/lib.rs

```diff
@@ -157,8 +183,10 @@ pub mod pallet {
 						"Cannot vest less than the existential deposit"
 					);
 					scheduled_vests.push((account.clone(), amount.clone()));
+					unvested_issuance = unvested_issuance.saturating_add(*amount);
 				}
 				VestingSchedule::<T>::insert(time, scheduled_vests);
+				TotalUnvestedIssuance::<T>::set(unvested_issuance);
```

Comment:
Noticed that the api has both `put` and `set` but it's unclear to me what the difference is.